app/lib/main.dart

```diff
@@ -75,6 +80,12 @@ Future<void> _startAppInner(Widget app, bool withSentry) async {
     child: app,
   );
 
+  if (withMatomo) {
```

Comment:
doesn't this mean we always are opt-in at first? this is against regulations, we might be opt-out first. Please add
```suggestion
  if (withMatomo) {
     MatomoTracker.instance.setOptOut(true);
```

That said... where/when are we reading the setting at startup?